content/guides/saml.md

```diff
@@ -0,0 +1,108 @@
+* If **sn** and **givenName** are provided, they will be used to update the user’s name in their Datadog profile. 
+
+## Additional Features Available On Request
+The following features are available on request. If you would like access to them, contact support and describe your scenario. The feature will be available on  your account after some configurations on our side.
+
+### Just in Time Provisioning (JIT Provisioning)
+Normally users must be invited to Datadog, even for organizations with SAML enabled. If a user that has not been invited to Datadog logs in via an Org's IdP, the SAML assertion will be validated, but they will be redirected to a SAML Error page.
+
+Some organizations might not want to have to invite all of their users to Datadog. If the organization has SAML enabled, and the saml_autocreate_user feature flag is enabled, Datadog will automatically create a user in response to a valid SAML assertion that contains a user handle that doesn't exist yet. This way anyone with a valid login on the organization's IdP can use Datadog without needing an invite.
+
+It is up to the organization to configure their IdP to not send assertions to Datadog if they don't want a particular user to access Datadog.
+
+### IdP Initiated Login
+
+The normal workflow is that when the Datadog url is loaded, the browser is redirected to the customer IdP, user types in credentials, then the IdP redirects back to Datadog. Some IdPs have the ability to send an assertion directly to Datadog without first getting an AuthnRequest (IdP Initiated Login). 
+
+In the normal setup, we won't know which org the assertion came from and this will result in an error page with a message saying that SAML Response is missing "InResponseTo" attribute. 
+
+After enabling the feature (and waiting for caches to clear) the customer will need to get a new version of the SP Metadata, which will have a different, org-specific AssertionConsumerService endpoint to send assertions to.
+
+### SAML Strict Mode
+For an org that has the saml_strict_mode feature flag enabled, users can not log in using a username and password on the login page - they must log in via their org's SAML IdP.
+
+Normally users who are invited to an org using SAML will not have a password, so they can't use the login page anyway. However, there are possibilities where a user at a SAML org can have a password
+
+* The user was created before the Org enabled SAML
+* Support added a temporary password and the user reset it In these cases (or in any case where a user has a password in the postgres database) a user can log in either with SAML (via the IdP) or with the login page using their Datadog username and password.
```

Comment:
I'd suggest collapsing the first bullet into the paragraph and eliminating the second bullet:

Normally users who are invited to an org using SAML will not have a password, so they can’t use the login page anyway. However, there are possibilities where a user at a SAML org can have a password, for instance if the user was created before the org enabled SAML.

For the IdP Initiated Login section, the following might be more clear (though I am not sure 'Customer' and 'Org' are appropriate words to use in documentation):

When Datadog is configured for SP Initiated Login (which is the default), upon loading an org's "Single Sign On Login URL" the browser is redirected to the customer IdP with a SAML AuthnRequest. On the customer IdP the user types in credentials. If the credentials are correct the IdP redirects back to Datadog with a SAML Assertion.  Before validating the SAML Assertion, Datadog checks to see if the SAML Assertion was issued in response to a SAML AuthnRequest issued by Datadog.

Some IdPs have the ability to send an assertion directly to Datadog without first getting an AuthnRequest, which is referred to as IdP Initiated Login.

If Datadog is configured for SP Initiated Login (which is the default) and receives an assertion from an IdP trying to initiate and  IdP Initiated Login session, Datadog will not be able to match the Assertion with an AuthnRequest (since no AuthnRequest was made).  This will result in an error page with a message saying that SAML Response is missing “InResponseTo” attribute.

Datadog support can configure IdP Initiated Login for an org.  After IdP Initiated Login is enabled, the customer the customer will need to get a new version of the SP Metadata, which will have a different, org-specific AssertionConsumerService endpoint to send assertions to.